config/config.exs

```diff
@@ -26,9 +44,9 @@ config :logger, :console,
 # Use Jason for JSON parsing in Phoenix
 config :phoenix, :json_library, Jason
 
-# LaunchDarkly SDK key
-config :hello_phoenix, ld_sdk_key: System.get_env("LD_SDK_KEY")
-
 # Import environment specific config. This must remain at the bottom
 # of this file so it overrides the configuration defined above.
-import_config "#{Mix.env()}.exs"
+import_config "#{config_env()}.exs"
+
+# LaunchDarkly SDK key
```

Comment:
This line down was added. Same as the original project. Everything above was generated from the template.